tests/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM dockerswarm/dind:1.8.1
+MAINTAINER Joffrey F <joffrey@docker.com>
+
+RUN mkdir /home/docker-py
+WORKDIR /home/docker-py
+
+RUN apt-get update
+RUN apt-get install -y python python-setuptools && easy_install pip
```

Comment:
> best practices says to make these a single line, otherwise you can run into a situation where the update is cached without the available package you need to install

WIll update!

> instead of easy_install pip why not add python-pip to the apt-get install line? If you want a newer version https://pip.pypa.io/en/latest/installing.html#install-pip suggests their install script.

"Surprisingly" (har har), python-pip [is outdated/broken](http://stackoverflow.com/questions/27341064/how-do-i-fix-importerror-cannot-import-name-incompleteread), so I'm using `easy_install` to get the latest version. I guess I could fetch their install script and use that instead, but that seems really unnecessary for what we're doing.